src/bindgen/ir/enumeration.rs

```diff
@@ -492,6 +496,10 @@ impl Item for Enum {
         }
     }
 
+    fn generic_params(&self) -> Option<&GenericParams> {
```

Comment:
We could have a `const EMPTY: Self = Self(Vec::new());` in `GenericParams` to make this a bit simpler perhaps (remove the `Option`, make callers return `&GenericParams::EMPTY` instead)?

How does that sound?